packages/php/email-editor/phpcs.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset name="Custom WordPress Standards">
+	<!-- Set the base standard to WordPress -->
+	<rule ref="WordPress"/>
+
+	<!-- Define files and folders to scan -->te
```

Comment:
`te` here seems to be some leftover. Isn't it?

Reply:
Nice catch! It's addressed in a new commit.